snare/server.py

```diff
@@ -71,18 +69,22 @@ async def handle_request(self, request):
             if previous_sess_uuid is None or not previous_sess_uuid.strip() or previous_sess_uuid != cur_sess_id:
                 headers.add("Set-Cookie", "sess_uuid=" + cur_sess_id)
 
+        if status_code == 404:
+            raise web.HTTPNotFound(headers=headers)
+
         return web.Response(body=content, status=status_code, headers=headers)
```

Comment:
Special if-check for status code 404 because returning `web.Response` with `status_code=404` did not work as intended.